cmd/zrepl/data_conn.c

```diff
@@ -116,7 +120,7 @@ uzfs_zvol_socket_write(int fd, char *buf, uint64_t nbytes)
 	while (nbytes) {
 		count = write(fd, (void *)p, nbytes);
 		if (count <= 0) {
```

Comment:
not related to this PR.. got to see this now..
can you change this to 'count < 0' as 0 is valid response for write call?

Reply:
good catch. We still need to treat zero return code differently as it means that connection was closed and we need to break out from the loop, however we should not be printing errno, but print "connection closed" message instead. The same applies to read() call above. I will fix both.

Reply:
oh, read() call is handled correctly. So only fixing the write path.

Reply:
right.. so, below added code will be removed right @jkryl ? and also condition can be changed to 'count < 0'